src/App.js

```diff
@@ -1,19 +1,21 @@
 import { Route,Routes } from "react-router-dom";
 import "./App.css";
-import {Home,History,WatchLater,Liked,ErrorPage,Videolist, LoginPage, VideoPage } from "./pages/index";
+import {Home,History,WatchLater,Liked,ErrorPage,Videolist, LoginPage, VideoPage, Playlist, PlaylistPage } from "./pages/index";
 
 function App() {
   return (
     <>
-      <Routes>
+     <Routes>
      <Route path="/" element={ <Home/> }></Route> 
      <Route path="/signup" element={<LoginPage />} ></Route>
      <Route path="/history" element={ <History />} ></Route>
      <Route path="/watchlater" element={<WatchLater />} ></Route>
-     <Route path="/playlists" element={<WatchLater />} ></Route>
-     <Route path="/explore" element={<Videolist />} ></Route>
+     <Route path="/playlists" element={<Playlist />} ></Route>
+     <Route path="/playlists/:playlistId" element={<PlaylistPage />} ></Route>
+     {/* <Route path="/explore" element={<Videolist />} ></Route> */}
      <Route path="/liked" element={<Liked />} ></Route>
-     <Route path="/explore/:id" element={<VideoPage />} ></Route>
+     <Route path="/explore/:categoryName" element={<Videolist />} ></Route>
+     <Route path="/explore/video/:id" element={<VideoPage />} ></Route>
      <Route path="*" element={<ErrorPage />} ></Route>
     </Routes>   
     </>  
```

Comment:
Instead of App.js, you can have all your routes in some other single file and just export that file in App.js

Reply:
Hey Rajat, thanks for pointing that out. I will make these changes at the end.